learn/gameplay/summoning.md

```diff
@@ -41,4 +43,15 @@ The act of summoning is an exhausting process, and that exhaustion compounds wit
 
-What’s this? Yes, unfortunately only Gen0 heroes are able to maintain their attunement to Gaia summon after summon. All later generations simply have a limit to how many times they can focus their energies into Gaia’s Tears. Once they have expended the last of their attunement energies, they can no longer summon. Another quirk of the summoning process is that the portal seeks those who have similar Gaia attunement strength as the weakest summoner. So if a gen0 summoner hero infuses a crystal with a gen1 summoner hero with only four remaining summons worth of attunement, then the summoned gen2 will only be able to summon three times (the same number of remaining summons as the weakest summoner after summoning this new hero). Two gen0 summoners will summon a gen1 with ten summons worth of attunement. Rumors say that some more advanced heroes have additional limitations and Gaia's Tear costs when it comes to summoning, though none have been found to discover what those limits and costs might be...
+What’s this? Yes, unfortunately only Gen0 heroes are able to maintain their attunement to Gaia summon after summon. All later generations simply have a limit to how many times they can focus their energies into Gaia’s Tears. Once they have expended the last of their attunement energies, they can no longer summon. Another quirk of the summoning process is that the portal seeks those who have similar Gaia attunement strength as the weakest summoner. So if a gen0 summoner hero infuses a crystal with a gen1 summoner hero with only four remaining summons worth of attunement, then the summoned gen2 will only be able to summon three times (the same number of remaining summons as the weakest summoner after summoning this new hero). Two gen0 summoners will summon a gen1 with ten summons worth of attunement. 
+
+Higher ranked heroes have begun to arrive at the kingdom!!!
+
+![Summoner](https://user-images.githubusercontent.com/91647016/136327660-f398e0a4-cb99-4be5-b102-8a20b9366053.png)
+
+Heroes of higher rank have better starting stats and stat growth parameters than those of ranks below them. However, a side effect of their advanced abilities is that they require more Gaia's tears to perform a summon than heroes of lower ranks. For example, the summoner depicted above must pay 40 tears to be able to summon. In addition, higher ranked heroes are more difficult for the portal to have a chance to locate, so higher rank heroes have lower maximum summons than lower ranked ones. This can be seen in the image of the Summoner above. 
```

Comment:
Should "Gaia's tears" here be capitalized for consistency with the rest of the doc?